security/cmd/node_agent_k8s/main.go

```diff
@@ -222,6 +248,10 @@ var (
 	secretRotationIntervalEnv          = env.RegisterDurationVar(SecretRotationInterval, 10*time.Minute, "").Get()
 	staledConnectionRecycleIntervalEnv = env.RegisterDurationVar(staledConnectionRecycleInterval, 5*time.Minute, "").Get()
 	initialBackoffEnv                  = env.RegisterIntVar(InitialBackoff, 10, "").Get()
+	monitoringPortEnv                  = env.RegisterIntVar(MonitoringPort, 9093,
+		"The port number for monitoring Citadel agent").Get()
+	enableProfilingEnv = env.RegisterBoolVar(EnableProfiling, false,
```

Comment:
Any reason not to default to true? Pilot does. Pretty useful if you run into issues. For instance if you had a memory leak you would have to restart citadel to profile it and now you lose all the state.
Great to see this added thogh